collector/pg_shared_preload_libraries.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2024 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collector
+
+import (
+	"context"
+	"database/sql"
+	"sort"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const sharedPreloadLibrariesSubsystem = "shared_preload_libraries"
```

Comment:
```suggestion
const sharedPreloadLibrariesSubsystem = "settings_shared_preload_libraries"
```

If we're only ever going to emit a metric from querying `pg_settings` I think it's worth either applying that namespace in the metric constant, or in the `namespace` that's getting passed below, which I'm not sure where that comes from. Follow your heart.